README.md

```diff
@@ -1,27 +1,87 @@
-# reverse_proxy
+# network_management
+
+This role configures your network interfaces automatically. This role also support complex setups with openvswitch and different vlans. On debian based systems a proper `/etc/network/interfaces` configuration is generated unless `network_management_always_script` is set to `True`.
 
 
 ## Requirements
 
+A Linux distribution with debian networking support or systemd as init system.
+`python-netaddr` has to be installed on the executing machine.
 
 ## Role Variables
-```yml
-```
-Default values are specified [here](defaults/main.yml) (except auth, which is specified by template file).
 
-## Dependencies
+### Primary
+| Option                                    | Type                         | Default                  | Description                                                                                                                                        | Required |
+|-------------------------------------------|------------------------------|--------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------|:--------:|
+| `network_management_always_script`        | boolean                      | `False`                  | Always generate a systemd service and network management script (overwrites networking.service if it exists)                                       |     N    |
```

Comment:
I don't see a script being generated

Reply:
Still don't see it. Implement it or remove the option as well as documentation of it.